lib/ansible/modules/cloud/amazon/rds_instance.py

```diff
@@ -0,0 +1,1236 @@
+        creation_source=dict(choices=['snapshot', 's3', 'instance']),
+        force_update_password=dict(type='bool', default=False),
+        promote=dict(type='bool', default=False),
+        purge_cloudwatch_logs_exports=dict(type='bool', default=True),
+        purge_tags=dict(type='bool', default=True),
+        wait=dict(type='bool', default=True),
+    )
+
+    parameter_options = dict(
+        allocated_storage=dict(type='int'),
+        allow_major_version_upgrade=dict(type='bool'),
+        apply_immediately=dict(type='bool', default=False),
+        auto_minor_version_upgrade=dict(type='bool'),
+        availability_zones=dict(type='list', aliases=['zones', 'az']),
+        backup_retention_period=dict(type='int'),
+        ca_certificate_identifier=dict(),
+        character_set_name=dict(),
+        copy_tags_to_snapshot=dict(type='bool'),
+        db_cluster_identifier=dict(aliases=['cluster_id']),
+        db_instance_class=dict(aliases=['class']),
+        db_instance_identifier=dict(required=True, aliases=['instance_id', 'id']),
+        db_name=dict(),
+        db_parameter_group_name=dict(),
+        db_security_groups=dict(),
+        db_snapshot_identifier=dict(),
+        db_subnet_group_name=dict(),
+        domain=dict(),
+        domain_iam_role_name=dict(),
+        enable_cloudwatch_logs_exports=dict(type='list'),
+        enable_iam_database_authentication=dict(type='bool'),
+        enable_performance_insights=dict(type='bool'),
+        engine=dict(choices=[
+            "aurora",
+            "aurora-mysql",
+            "aurora-postgresql",
+            "mariadb",
+            "mysql",
+            "oracle-ee",
+            "oracle-se",
+            "oracle-se1",
+            "oracle-se2",
+            "postgres",
+            "sqlserver-ee",
+            "sqlserver-ex",
+            "sqlserver-se",
+            "sqlserver-web",
+        ]),
+        engine_version=dict(),
+        final_db_snapshot_identifier=dict(aliases=['final_snapshot_identifier']),
+        force_failover=dict(type='bool'),
+        iops=dict(type='int'),
+        kms_key_id=dict(),
+        license_model=dict(choices=['license-included', 'bring-your-own-license', 'general-public-license']),
+        master_user_password=dict(aliases=['password'], no_log=True),
+        master_username=dict(aliases=['username']),
+        monitoring_interval=dict(),
+        monitoring_role_arn=dict(),
+        multi_az=dict(type='bool'),
+        new_db_instance_identifier=dict(aliases=['new_instance_id', 'new_id']),
+        option_group_name=dict(),
+        performance_insights_kms_key_id=dict(),
+        performance_insights_retention_period=dict(),
+        port=dict(type='int'),
+        preferred_backup_window=dict(aliases=['backup_window']),
+        preferred_maintenance_window=dict(aliases=['maintenance_window']),
+        processor_features=dict(type='dict'),
+        promotion_tier=dict(),
+        publically_accessible=dict(type='bool'),
```

Comment:
I think `publicly` is the more common spelling for this https://en.wiktionary.org/wiki/publically

Reply:
Oops. Thanks for catching that. That's even a bug considering it doesn't match the expected CamelCase and won't be applied to any RDS instances.